src/io/flutter/view/DiagnosticsTreeCellRenderer.java

```diff
@@ -182,28 +180,22 @@ else if (isLinkedChild || panel.currentShowNode == value) {
     }
     else {
       // Non property, regular node case.
-      if (StringUtils.isNotEmpty(name) && node.getShowName()) {
-        // color in name?
-        if (name.equals("child") || name.startsWith("child ")) {
+      if (StringUtils.isNotEmpty(name) && node.getShowName() && !name.equals("child")) {
+        if (name.startsWith("child ")) {
           appendText(name, SimpleTextAttributes.GRAYED_ATTRIBUTES);
         }
         else {
           appendText(name, textAttributes);
         }
 
         if (node.getShowSeparator()) {
-          // Is this good?
           appendText(node.getSeparator(), SimpleTextAttributes.GRAY_ATTRIBUTES);
         }
         else {
           appendText(" ", SimpleTextAttributes.GRAY_ATTRIBUTES);
         }
       }
 
-      if (panel.isCreatedByLocalProject(node)) {
```

Comment:
change this to 
if (panel.detailsSubtree && panel.isCreatedBylocalProject(node) && !panel.isHighlightNodesShownInBothTrees) {
  textAttributes = textAttributes.derive(SimpleTextAttributes.REGULAR_BOLD_ATTRIBUTES.getStyle(), null, null, null);
}|

that way it is still clear which nodes in the details tree are created by the local project but the entire summary tree is not bold.

Reply:
gotcha, will do